tagging.py

```
@@ -1,16 +1,503 @@
 import re
-from word2number import w2n
-from dotenv import load_dotenv
+from enum import Enum, auto
+from typing import Callable, Protocol
 
-api_key = os.getenv("API_KEY")
+#==================================
+#   Filname Lexing
+#==================================
 
+def is_numeric_or_digit(x: str) -> bool:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)